src/routes/index.svelte

```diff
@@ -73,23 +50,82 @@
     let loading = false
     let highlightedSymbolIndex = null
 
+    let isDocumentReady = false
+
     onMount(() => {
+        let DEFAULT_OPTIONS = []
+
+        if (typeof document !== 'undefined') {
+            isDocumentReady = true
+        }
+
+        selectedPriceOption = getDefaultPriceOption(price)
+
+        DEFAULT_OPTIONS = getDefaultOptions(symbols)
         fetchStockData(DEFAULT_OPTIONS)
     })
 
+    const getDefaultOptions = symbols => {
+        let symbolList
+        const storedSymbols = localStorage.getItem(LS_SYMBOLS)
+
+        if (symbols) {
+            symbolList = symbols.split(',')
+        } else if (storedSymbols) {
+            const symbolsString = storedSymbols
+            symbolList = symbolsString.split(',')
+        }
+
+        if (symbolList && symbolList.length > 0) {
```

Comment:
You can declare `let symbolList = []` and remove `symbolList &&` from this if statement